src/frontend/views/games_view.py

```diff
@@ -45,8 +45,8 @@ def get_context_data(self, **kwargs):
       return context
 
 
-class GamesListView(ListView):
-  template_name = "pages/games_list.html"
+class GamesListByNetworkView(ListView):
```

Comment:
Maybe we should split the views into a directory of views ?

Reply:
What do you mean? What is a directory of views?